catt/cli.py

```diff
@@ -180,10 +180,11 @@ def cli(ctx, device):
 @click.option(
     "-y",
     "--ytdl-option",
+    "--ytdlp-option",
```

Comment:
I don't think this needs to be changed, and it would break a bunch of stuff. It can stay as it was, I think.

Reply:
Agreed

Reply:
Can you change it back though?

Reply:
```suggestion
    "--ytdl-option",
```